docs/framework/data/adonet/sql/linq/downloading-sample-databases.md

```diff
@@ -1,50 +1,48 @@
 ---
 title: "Downloading Sample Databases"
-ms.date: "03/30/2017"
+ms.date: "10/18/2018"
 ms.assetid: ef9d69a1-9461-43fe-94bb-7c836754bcb5
 ---
-# Downloading Sample Databases
-A number of samples and walkthroughs in the [!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)] documentation use the Northwind sample database and SQL Server Express Edition. You can download these products free of charge from the Microsoft download site.  
+# Download the Sample Databases, SQL Server Express, and tools
+A number of samples and walkthroughs in the [!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)] documentation use the Northwind sample database and SQL Server Express. You can download these products free of charge from Microsoft.
+
+## Download the Sample Databases
+
+### To download the sample databases
```

Comment:
if you don't add other databases here, you should just remove this redundant heading or add an intro paragraph